pkg/kubectl/rolling_updater.go

```diff
@@ -678,7 +680,9 @@ func AddDeploymentKeyToReplicationController(oldRc *api.ReplicationController, c
 	// Clean up any orphaned pods that don't have the new label, this can happen if the rc manager
 	// doesn't see the update to its pod template and creates a new pod with the old labels after
 	// we've finished re-adopting existing pods to the rc.
-	podList, err = client.Pods(namespace).List(labels.SelectorFromSet(selectorCopy), fields.Everything(), unversioned.ListOptions{})
+	selector = labels.SelectorFromSet(selectorCopy)
```

Comment:
Out of curiosity, did you automate this kind of changes? If so, how did you do that?